media/js/src/assetDetail/AssetDetail.jsx

```diff
@@ -581,6 +588,20 @@ export default class AssetDetail extends React.Component {
                 this.startButtonRef.current.focus();
             }
         }
+
+        if (prevState.isDrawing !== this.state.isDrawing) {
```

Comment:
I'm seeing this work the first time through, but then reappearing in subsequent "draw" events. Let's talk through the desired flow here tomorrow morning.